.github/workflows/test-gh.yml

```diff
@@ -23,6 +23,16 @@ jobs:
       uses: actions/checkout@v1
       with:
         path: src/github.com/${{ github.repository }}
+    - name: Install Carvel Tools
+      uses: vmware-tanzu/carvel-setup-action@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        only: ytt, kapp, kbld, imgpkg, vendir
```

Comment:
dont think we need imgpkg or vendir as binaries.

Reply:
There is a non e2e test for vendir that relies on vendir being installed: https://github.com/vmware-tanzu/carvel-kapp-controller/blob/53cbfee531155c40268ff8914f62becd15a94836/pkg/app/app_reconcile_test.go#L21

I could also see imgpkg being used at some point in our testing, but I can remove for now as it's not currently used.

Reply:
ah sounds good.